public/docs/getting_started/client.md

```diff
@@ -1,20 +1,54 @@
+# Choosing an Ethereum client
+
 There are many Ethereum clients to choose from. We recommend using different clients when developing and deploying.
 
-# When Developing
+## Developing locally
+
+### Truffle Develop
+
+We recommend using Truffle Develop, an Ethereum client built directly into Truffle which contains an in-memory blockchain that runs only on your development machine.
+
+It processes transactions instantly instead of waiting for the default block time, so you can test that your code works quickly, and it tells you immediately when your smart contracts run into errors. It also makes a great client for automated testing.
+
+Because Truffle Develop is built into Truffle, no external installation is required. To run Truffle Develop, type the following in a terminal:
+
+```shell
+truffle develop
+```
+
+This will run the client on `http://localhost:9545`. it will display the first 10 accounts, and a deterministic mnemonic for the blockchain. Truffle Develop uses the same mnemonic upon every launch:
+
+```
+candy maple cake sugar pudding cream honey rich smooth crumble sweet treat
+```
+
+Once the Truffle Develop console is launched, you can run all of the commands you are used to, but omitting the `truffle` prefix. So, for example, to compile your smart contract, instead of typing `truffle compile` on the terminal, just type `compile` in the Truffle Develop console.
```

Comment:
"Truffle Develop console" is being introduced here as a new thing. (i.e., to a new user, it might create the question, is there a difference between "Truffle Develop" and "Truffle Develop console"?). Additionally, "all the commands you're used to" doesn't mean anything to first time readers. I'd rephrase like this:

> Once launched, Truffle Develop will provide you with a console you can use to run all available Truffle commands. These commands are input by omitting the `truffle` prefix. So, for example, to compile your smart contracts, instead of typing `truffle compile`, you need to only type `compile`.

Reply:
Just to make sure, I know Truffle Develop is the application, but is it correct to refer to the thing we types commands at as the "Truffle Develop console"? I'd like to be able differentiate the thing from the application as a whole.

Reply:
That's sounds good to me, in general. Here, the console hasn't yet been introduced (i.e., a new user may confuse the two) so I wanted to ensure people understand that `truffle develop` creates a console.

Additionally (new thought) we should link to [this page](https://github.com/trufflesuite/trufflesuite.com/blob/master/public/docs/getting_started/console.md) on the site in (or near) this paragraph so people can read more on how to use the console.